include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -638,6 +622,23 @@ constexpr function_wrapper_t alignment_configurator::configure_free_ends_initial
 
         // Make initialisation policy a deferred CRTP base and delegate to configure the find optimum policy.
         using gap_init_policy_t = deferred_crtp_base<affine_gap_init_policy, policy_trait_type>;
+
+        if constexpr (traits_t::is_global && !config_t::template exists<seqan3::align_cfg::aligned_ends>())
+        {
+            auto method_global_cfg = cfg.get_or(align_cfg::method_global{});
+
+            if (method_global_cfg.free_end_gaps_sequence1_leading ||
+                method_global_cfg.free_end_gaps_sequence1_trailing ||
+                method_global_cfg.free_end_gaps_sequence2_leading ||
+                method_global_cfg.free_end_gaps_sequence2_trailing)
+            {
+                // append an extra aligned_ends configuration for static configuration that is needed in
+                // configure_free_ends_optimum_search (within select_find_optimum_policy)
+                auto ends_config = seqan3::align_cfg::aligned_ends{seqan3::free_ends_none};
+                return configure_free_ends_optimum_search<function_wrapper_t, policies_t..., gap_init_policy_t>(cfg | ends_config);
+            }
+        }
+
```

Comment:
```suggestion
```

I tried this without it and it still compiled and passed all tests, are you sure that you still need this?

Reply:
Did it?? :laughing: Damn... I thought I had some test failures because of this...
And I thought Rene told me we still need the static distinction in some simd policy :shrug: But if all tests run through (and we will get rid of the old alignment code anyway) I guess I'll just remove it.